florestad/src/florestad.rs

```diff
@@ -326,7 +326,7 @@ impl Florestad {
         // For now we only have compatible bridges on signet
         let pow_fraud_proofs = match self.config.network {
             crate::Network::Bitcoin => false,
-            crate::Network::Signet => true,
+            crate::Network::Signet => false,
```

Comment:
Why `false` now?

Reply:
This was meant to be temporary :joy: . Nice catch!

Reply:
Making this `true` again is missing from your push

Reply:
Shoot, you're right.

I think now I got it

Reply:
Yes!